Integrations/src/main/java/io/deephaven/integrations/learn/Computer.java

```diff
@@ -0,0 +1,62 @@
+package io.deephaven.integrations.learn;
+
+import org.jpy.PyObject;
+
+/**
+ * Instantiates the objects needed for computation using the given indices.
+ */
+public class Computer {
+
+    private final PyObject func;
+    private final int batchSize;
+    private final Input[] inputs;
+    private Future current;
+    private int offset;
+
+    /**
+     * Creates a new Computer.
+     *
+     * @param func Function for performing a computation on these inputs.
+     * @param inputs Inputs to this function.
+     * @param batchSize Maximum number of rows for deferred computation.
+     */
+    public Computer(PyObject func, int batchSize, Input ... inputs) {
+
+        if (batchSize <= 0) {
+            throw new IllegalArgumentException("Max size must be a strictly positive integer."); }
+
+        if (inputs.length == 0) {
+            throw new IllegalArgumentException("Cannot have an empty input list."); }
```

Comment:
Same kind of questions as my other comments on input validation. Since the user doesn't handle this, is verifying that Input itself is correct at an earlier stage sufficient to guarantee that this will also be correct?
Null function is now taken care of.